src/accelerate/data_loader.py

```diff
@@ -340,8 +340,10 @@ class DataLoaderShard(DataLoader):
             - `"cuda"`: the CUDA random number generator (GPU only)
             - `"xla"`: the XLA random number generator (TPU only)
             - `"generator"`: an optional `torch.Generator`
-        generator (`torch.Generator`, *optional*):
+        synchronized_generator (`torch.Generator`, *optional*):
             A random number generator to keep synchronized across processes.
+        split_batches (`int`, *optional*, defaults to 0):
+            The number of batches to skip at the beginning.
```

Comment:
```suggestion
        skip_n_batches (`int`, *optional*, defaults to 0):
            The number of batches to skip at the beginning.
```

I think having `skip_n_batches` instead would be better on the user as it's more descriptive.

Reply:
Not really fan of the longer name, and those are not really public classes anyway (in the sense that they are not in the init) so will keep it as is.